saleor/order/views.py

```diff
@@ -115,8 +115,7 @@ def start_payment(request, order, gateway):
     ctx = {
         'form': form, 'payment': payment,
         'client_token': client_token, 'order': order}
-    return TemplateResponse(
-        request, [template, 'order/payment/default.html'], ctx)
+    return TemplateResponse(request, template, ctx)
```

Comment:
What's the reason about dropping `order/payment/default.html`? It was really useful for gateways using modals, could we restore that?

Reply:
*Resolved.*